src/hotspot/cpu/x86/x86.ad

```diff
@@ -6564,10 +6564,10 @@ instruct vector_uminmaxq_reg(vec dst, vec a, vec b, vec xtmp1, vec xtmp2) %{
   ins_pipe( pipe_slow );
 %}
 
-instruct vector_uminmax_reg_masked(vec dst, vec src1, vec src2, kReg mask) %{
-  match(Set dst (UMinV (Binary src1 src2) mask));
-  match(Set dst (UMaxV (Binary src1 src2) mask));
-  format %{ "vector_uminmax_masked $dst, $src1, $src2, $mask\t! umin/max masked operation" %}
+instruct vector_uminmax_reg_masked(vec dst, vec src2, kReg mask) %{
```

Comment:
Should `src2` be renamed to `src`?

Reply:
For predicated vector operations, we either populate destination vector lane with the result of the operation if the corresponding mask bit is true or else retain the original contents of lanes.
`vec1.lanewise(VectorOperators.UMIN, vec2) 
`
Here, UMinVNode (vec1, vec2) IR has two source inputs, and two addr matcher pattern alias the first source and destination operand. So src2 looks appropriate.

Reply:
Thanks for the clarification, makes sense.